### PR TITLE
Register csur.is-a.dev

### PR DESCRIPTION
**Pull request by ocoke**
Added `csur.is-a.dev` using the [CLI](https://www.npmjs.com/package/@is-a-dev/cli).

### CODE REVIEW

**Comment by ocoke** [comment]
CranSurvey is an open-source tool for collecting survey data, it is written in Vue & TypeScript, and `csur` is the short of it.

The page file of the site is in [ocoke/csur-site](https://github.com/ocoke/csur-site), the main code of CranSurvey is in [ocoke/CranSurvey](https://github.com/ocoke/CranSurvey).

Thank you.

**Review by priyazsh — COMMENTED** [review]
is-a-dev doesn't support vercel

**Comment by ocoke** [comment]
well, let me change the record to netlify later, thx.

**Comment by oyepriyansh** [comment]
They don't support both Vercel and Netlify due to TXT verification. Instead, you can use Cloudflare Pages.

File: domains/csur.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "ocoke",
+        "email": "awa@outlook.ie"
+    },
+
+    "record": {
+        "CNAME": "csur-site.netlify.app"
```
Comment (DEV-DIBSTER) [inline domains/csur.json:8]: We do not support netlify.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Pull request closed without merging**